libs/sdk-core/src/persist/sync.rs

```diff
@@ -201,78 +201,255 @@ impl SqliteStorage {
             [],
         )?;
 
+        // Sync remote swaps_fees table, which contains dynamic fees used in swaps
+        // created_at is used to settle conflicts, since we assume small variations in the client local times
+        Self::sync_swaps_fees_local(&tx)?;
+
         tx.commit()?;
         con.execute("DETACH DATABASE remote_sync", [])?;
 
         Ok(())
     }
+
+    /// Insert or update to local db all rows that have created_at larger than in the local
+    fn sync_swaps_fees_local(tx: &Transaction) -> Result<()> {
+        // Both DBs have a swaps_fees row for the same swap address
+        tx.execute(
+            "
+        INSERT OR REPLACE INTO sync.swaps_fees
+         SELECT
+          remote_sync.swaps_fees.bitcoin_address as bitcoin_address,
+          remote_sync.swaps_fees.created_at as created_at,
+          remote_sync.swaps_fees.channel_opening_fees as channel_opening_fees
+         FROM remote_sync.swaps_fees
+          LEFT JOIN sync.swaps_fees ON sync.swaps_fees.bitcoin_address = remote_sync.swaps_fees.bitcoin_address
+         WHERE
+          remote_sync.swaps_fees.created_at > sync.swaps_fees.created_at
+         ;",
+            [],
+        )?;
+
+        // The local DB doesn't have a swaps_fees row matching one or more entries in the remote
+        tx.execute(
+            "
+        INSERT OR REPLACE INTO sync.swaps_fees
+         SELECT bitcoin_address, created_at, channel_opening_fees FROM remote_sync.swaps_fees
+          WHERE
+           bitcoin_address NOT IN (SELECT bitcoin_address FROM sync.swaps_fees)
+         ;",
+            [],
+        )?;
```

Comment:
I think changing the where clause to this, potentially has less unintended consequences with comparing a datetime with null. And it makes the second query obsolete.

```suggestion
          sync.swaps_fees.created_at is null or remote_sync.swaps_fees.created_at > sync.swaps_fees.created_at
```

Reply:
(I selected too many lines to delete. 239-241 should not be deleted)

Reply:
Good point, that way the two queries can be merged into one.

Done in aa35a5f1d3e4760bca4d96000b21ade483fad458